lookout/style/format/feature_extractor.py

```diff
@@ -235,6 +235,74 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None) \
                  the corresponding `VirtualNode`-s and the parents mapping \
                  or None in case no features were extracted.
         """
+        parsed_files, node_parents, vnode_parents = self._parse_vnodes(files, lines)
+        xy = self._files_to_xy(parsed_files)
+        if xy is None:
+            return None
+        X, y, vnodes_y, vnodes, sibling_indices_list = xy
+        self._log.debug("Features shape: %s", X.shape)
+        self._log.debug("Labels shape: %s", y.shape)
```

Comment:
let's remove it. It is obvious from the `X.shape`

Reply:
The first dimension yes, but the second?

Reply:
it is 1-d array, so no second dimention